dev/io.openliberty.microprofile.telemetry.2.0.logging.internal.container_fat/fat/src/io/openliberty/microprofile/telemetry/logging/internal/container/fat/LoggingServletTest.java

```diff
@@ -0,0 +1,224 @@
+import com.ibm.websphere.simplicity.ShrinkHelper.DeployOptions;
+import com.ibm.websphere.simplicity.log.Log;
+
+import componenttest.annotation.ExpectedFFDC;
+import componenttest.annotation.Server;
+import componenttest.containers.SimpleLogConsumer;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+
+/**
+ *
+ */
+@RunWith(FATRunner.class)
+public class LoggingServletTest {
+
+    private static Class<?> c = LoggingServletTest.class;
+
+    @Server("TelemetryLogsServer")
+    public static LibertyServer server;
+
+    public static final String SERVER_XML_ALL_SOURCES = "allSourcesServer.xml";
+    public static final String SERVER_XML_TRACE_SOURCE = "traceSourceServer.xml";
+    public static final String SERVER_XML_FFDC_SOURCE = "FFDCSourceServer.xml";
+
+    private static final String[] EXPECTED_FAILURES = { "CWMOT5005W", "SRVE0315E", "SRVE0777E" };
+
+    @ClassRule
+    public static GenericContainer<?> container = new GenericContainer<>(new ImageFromDockerfile()
+                    .withDockerfileFromBuilder(builder -> builder.from(TestUtils.IMAGE_NAME)
+                                    .copy("/etc/otelcol-contrib/config.yaml", "/etc/otelcol-contrib/config.yaml"))
+                    .withFileFromFile("/etc/otelcol-contrib/config.yaml", new File(TestUtils.PATH_TO_AUTOFVT_TESTFILES + "config.yaml"), 0644))
+                    .withLogConsumer(new SimpleLogConsumer(LoggingServletTest.class, "opentelemetry-collector-contrib"))
+                    .withExposedPorts(4317);
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        TestUtils.trustAll();
+        WebArchive telemetryLogApp = ShrinkWrap
+                        .create(WebArchive.class, "MpTelemetryLogApp.war")
+                        .addPackage(
+                                    "io.openliberty.microprofile.telemetry.logging.internal.container.fat.MpTelemetryLogApp")
+                        .addAsManifestResource(new File("publish/resources/META-INF/microprofile-config.properties"),
+                                               "microprofile-config.properties");
+
+        ShrinkHelper.exportDropinAppToServer(server, telemetryLogApp,
+                                             DeployOptions.SERVER_ONLY);
+
+        server.addEnvVar("OTEL_EXPORTER_OTLP_ENDPOINT", "http://" + container.getHost() + ":" + container.getMappedPort(4317));
+
+        server.startServer();
+
+        //Read to run a smarter planet
+        server.waitForStringInLogUsingMark("CWWKF0011I");
+        server.setMarkToEndOfLog();
+
+    }
+
+    /*
+     * Ensures all messages printed in the messages.log are bridged over to the otlp container.
+     */
+    @Test
+    public void testBridgedLogs() throws Exception {
+
+        assertTrue(server.isStarted());
+        TestUtils.runApp(server, "logs");
+
+        //Allow time for the collector to receive and bridge logs.
+        TimeUnit.SECONDS.sleep(5);
```

Comment:
instead of doing an explicit sleep here, can't we check if the `container.getLogs()` is not null and contains the bridged app msg, with a timeout? This is to prevent the hard 5 secs wait, regardless, when it might be quicker.